src/main/java/com/obal/dominos/App.java

```diff
@@ -1,19 +1,102 @@
 package com.obal.dominos;
 
-import java.util.Scanner;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.LinkedList;
 
-/**
- * Hello world!
- *
- */
+class Domino {
+    byte[] values;
```

Comment:
A set might be better, as the order of the domino values is not important (eg [1,6] should be the same as [6,1])